app/views/helps/new.html.erb

```diff
@@ -22,7 +22,7 @@
          <div class="col-md-9">
            <div class="form-group">
              <%= form.label(:affiliation, "What is your Stanford affiliation/department?") %>
-             <%= form.text_field(:affiliation, class: "form-control") %>
+             <%= form.text_field(:affiliation, class: "form-control", value: "(ex: Student)") %>
```

Comment:
If it's of equal accessibility value, I'm on the fence about using an input field value here vs. a placeholder (which renders until the user starts typing). Thoughts?

Reply:
Placeholder is probably better. I missed that option, fixing.

Reply:
OK, cool!

Reply:
@mjgiarlo In this case, `placeholder` is perfect - and it works better for the screen reader as well! So double win, thanks!